tests/unit/s2n_openssl_x509_test.c

```diff
@@ -76,5 +79,130 @@ int main(int argc, char** argv)
         }
     }
 
+    /* s2n_openssl_x509_get_cert_info */
+    /* clang-format off */
+    struct {
```

Comment:
If clang was mutilating your lists, it's because the last field of each of your structs don't have trailing commas.